src/sentry/incidents/charts.py

```diff
@@ -9,11 +9,11 @@
 from sentry.api import client
 from sentry.api.base import logger
 from sentry.api.serializers import serialize
-from sentry.api.serializers.models.alert_rule import AlertRuleSerializer
-from sentry.api.serializers.models.incident import DetailedIncidentSerializer
 from sentry.api.utils import get_datetime_from_stats_period
 from sentry.charts import backend as charts
 from sentry.charts.types import ChartSize, ChartType
+from sentry.incidents.endpoints.incident import DetailedIncidentSerializer
```

Comment:
it looks like we're changing adding `incidents` (👍), but also changing `models` to `endpoints`. Could you explain that a little? just curious if it's meant to represent an inbound/outbound filter or if it's an existing convention or? (Thanks!)

Reply:
Ah!

wait thismight be an oops.
one moment